imports/pages/groups/profile/index.js

```diff
@@ -86,26 +92,38 @@ export const GROUP_MUTATION = gql`
 
 const withGroupMutation = graphql(GROUP_MUTATION, {
   props: ({ mutate }) => ({
-    addToGroup: (groupId, message, communicationPreference) => mutate({
-      variables: { groupId, message, communicationPreference },
-    }),
+    addToGroup: (groupId, message, communicationPreference) =>
+      mutate({
+        variables: { groupId, message, communicationPreference },
+      }),
   }),
 });
 
 const defaultArray = [];
-class TemplateWithoutData extends Component {
 
+// UTILITIES
+const getLeaders = group =>
+  group &&
+  group.members &&
+  group.members.filter(x => x.role.toLowerCase() === "leader");
+
+const isCurrentPersonLeader = (person, leaders) =>
+  person &&
+  Array.isArray(leaders) &&
+  leaders.filter(x => x.person.id === person.id).length;
+
```

Comment:
getLeaders and isCurrentPersonLeader changed. They still need to be exported and tested